Bottom: d6a62ad1dead7e506de4197029a2612dde20961f
Top:    7d9980e41fc84975900e2eb78268a2d4d4cea9b5
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-17 16:19:40 +0800

Refresh of vhost_net-introduce-tx

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index ef228fe..4fd7de0 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -501,8 +501,7 @@ static int get_tx_bufs(struct vhost_net *net,
 		goto err;
 
 	if (*in) {
-		vq_err(vq, "Unexpected descriptor format for TX: "
-			"out %d, int %d\n", *out, *in);
+		vq_err(vq, "Unexpected descriptor format for TX: out %d, int %d\n", *out, *in);
 		ret = -EFAULT;
 		goto err;
 	}
